### ISSUE TITLE
Debug parameters evaluated, but not consumed by logger

### ISSUE THREAD

**Issue by LiquidTheDangerous** [xvzc/SpoofDPI#168]
For example, at this point a string is created from a slice, which causes copying.

```go
log.Debug("[PROXY] request from ", conn.RemoteAddr(), "\n\n", string(pkt.Raw()))

```

It is clear that this is an insignificant optimization, compared to the execution time of the IO request, but I would like to get rid of such places if possible.

something like 

```go

log.isDebugEnabled() {
...
}

```

would be nice

Additionally, I want the request body to be logged only under the trace level. #141

**Comment by xvzc** [comment]
Hmm should we change the name of the option debug to log-level?
Or we can actually just simplify or remove the logging request body part which is not really necessary.

**Comment by LiquidTheDangerous** [comment]
@xvzc 
Yes, I think viewing the body of the request will not be useful. It can be removed
